static/ru/index.template.html

```
@@ -0,0 +1,1555 @@
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div class="modal fade" id="weaponTypeModalContusion" tabindex="-1" aria-labelledby="portraitLabel"
+		aria-hidden="true">
+		<div class="modal-dialog modal-dialog-centered">
+			<div class="modal-content">
+				<div class="modal-header">
+					<h5 class="modal-title" id="portraitLabel">👉 Орудие убийства: удар тупым предметом </h5>
```

HTMLHint (1.9.2)

[error] The id value [ portraitLabel ] must be unique.

(id-unique)